event/event.go

```diff
@@ -248,36 +248,41 @@ func (r *MessageSubscription) Serve(handler MessageHandler) error {
 				<-semaphore
 			}()
 
-			id, publishedTime := getMetadata(msg)
+			id, publishedTime := getMetadata(gocloudMsg)
+			msg := Message{
+				Body: gocloudMsg.Body,
+				Metadata: Metadata{
+					ID:            id,
+					PublishedTime: publishedTime,
+					Attributes:    gocloudMsg.Metadata,
+				},
+			}
 
 			defer func() {
 				if err := recover(); err != nil {
 					// 64KB, if it is good enough for Go's stdlib it is good enough for us :-)
 					const size = 64 << 10
 					buf := make([]byte, size)
 					buf = buf[:runtime.Stack(buf, false)]
-					slog.Error("panic: message subscription: handling message", "message", msg, "stack_trace", string(buf))
-					if msg.Nackable() {
-						msg.Nack()
+					slog.Error("panic: message subscription: handling message",
+						"error", err,
+						"message", msg.Body,
```

Comment:
```suggestion
						"message_body", msg.Body,
```

Or else we will lose the original message we passed on the log, the "panic: message subscription: handling message", that being the "main" message is nice since it is easier to filter.